src/checkstyle/checkstyleMain.xml

```diff
@@ -21,7 +21,6 @@
 		<module name="UnusedImports">
 			<property name="processJavadoc" value="true" />
 		</module>
-		<module name="AvoidStarImport"/>
```

Comment:
:+1: 

I originally removed it because error-prone seems to have the exact same check enabled by default, but it occurs to me just now that error-prone doesn't seem to run the same check on test sources (for whatever reason), so I'll happily re-enable this Checkstyle module. :)